.kitchen.yml

```diff
@@ -55,12 +52,18 @@ suites:
         server_debian_password: q654321
   - name: postgresql
     run_list:
+      - recipe[apt]
```

Comment:
Um.. shouldn't we use our `transportapi-apt` wrapper cookbook over the original `apt-cookbook`? Not sure what's the intent here but thought I'd raise it so we're sure to pick the right one.